lib/inferno/apps/cli/templates/lib/%library_name%/metadata.rb.tt

```diff
@@ -0,0 +1,18 @@
+require_relative 'version'
+
+module <%= module_name %>
+  class Metadata < Inferno::TestKit
```

Comment:
I'm not wild about `Metadata` as the class name. Maybe `TestKitMetadata` would be better?

Reply:
I like the `Metadata`, it makes the global name `SomethingTestKit::Metadata` which makes sense since the RubyGem itself is the Test Kit, `SomethingTestKit` is the namespace, and `Metadata` is really just metadata and maybe config.

If anything I would rename/alias `Inferno::TestKit` to `Inferno::TestKitMetadata`.